contracts/dao/voting/vesting-lp-vault/src/contract.rs

```diff
@@ -0,0 +1,324 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{
+    to_binary, Binary, Deps, DepsMut, Env, Fraction, MessageInfo, Response, Uint128,
+};
+use cw2::set_contract_version;
+use cwd_interface::voting::{
+    BondingStatusResponse, TotalPowerAtHeightResponse, VotingPowerAtHeightResponse,
+};
+use cwd_interface::Admin;
+
+use crate::state::{CONFIG, DAO};
+use neutron_lockdrop_vault::voting_power::get_voting_power;
```

Comment:
actually one more thing: I find it incorrect to keep this func in the `neutron_lockdrop_vault`. please move it to some common package and reuse in both vaults cause it's not lockdrop specific, is it?

Reply:
done!